lib/tests/2wp.js

```diff
@@ -777,6 +777,53 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should reject and refund a pegout when fee per kb is above value', async () => {
+
+      // Arrange
+
+      // Create a pegin for the sender to ensure there is enough funds to pegout and because this is the natural process
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const peginValueInSatoshis = btcToSatoshis(0.5);
+      const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, senderRecipientInfo.btcSenderAddressInfo, satoshisToBtc(peginValueInSatoshis));
+      await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
```

Comment:
Could this whole part maybe be moved to a method? `fundBridgeForPegout` or something of the sort. What do you think?

Reply:
Updated.